### PR TITLE
Add CoinDCX

### PR DESCRIPTION
**Pull request by archit11111**
CoinDCX has Alexa rank - 5408 and supports 2FA

### CODE REVIEW

File: img/c/coindcx.com.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	<path class="st2" d="M16.1,16.1c0-0.6-0.3-1.2-0.8-1.5L5.4,8.9c0.1,3.3,1.9,6.2,4.7,7.9l4.4,2.6c0.7,0.4,0.6,1.5-0.1,1.9l1.1-0.6
+		c0.3-0.2,0.6-0.6,0.6-1v-1.3C16,18.3,16.1,16.1,16.1,16.1z"/>
+	<path class="st3" d="M14.5,19.4l-1.8-1c0,0,1.1,1,1.5,1.4c0.4,0.5,0.1,1.5,0.1,1.5C15,20.9,15.2,19.8,14.5,19.4z"/>
+	<path class="st1" d="M22.3,12.2c-0.4,1.1-1.2,2-2.2,2.5l-1.8-1c-0.7-0.4-0.6-1.5,0.1-1.9L22,9.8c0.2-0.1,0.5,0,0.6,0.2c0,0,0,0,0,0
+		c0,0,0,0.1,0,0.2C22.6,10.8,22.5,11.6,22.3,12.2z"/>
+	<path class="st2" d="M16.7,17c0,0.6,0.3,1.2,0.8,1.5l9.9,5.7c-0.1-3.3-1.9-6.2-4.7-7.9l-4.4-2.6c-0.7-0.4-0.6-1.5,0.1-1.9l-1.1,0.6
+		c-0.3,0.2-0.6,0.6-0.6,1v1.3C16.7,14.8,16.7,17,16.7,17z"/>
+	<path class="st3" d="M18.3,13.8l1.8,1c0,0-1.1-1-1.5-1.4c-0.4-0.5-0.1-1.5-0.1-1.5C17.7,12.2,17.6,13.3,18.3,13.8z"/>
+</g>
+</svg>
```
Comment (Carlgo11) [inline img/c/coindcx.com.svg]: Here's a minimized version
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 32 32"><circle cx="16" cy="16" r="16" fill="#fff"/><path d="M10.5 20.9c.4-1.1 1.2-2 2.2-2.5l1.8 1c.7.4.6 1.5-.1 1.9l-3.6 2.1c-.2.1-.5 0-.6-.2V23c-.1-.7 0-1.4.3-2.1z"/><path d="M16.1 16.1c0-.6-.3-1.2-.8-1.5L5.4 8.9a9.6 9.6 0 0 0 4.7 7.9l4.4 2.6c.7.4.6 1.5-.1 1.9l1.1-.6c.3-.2.6-.6.6-1v-3.6z"/><path d="m14.5 19.4-1.8-1 1.5 1.4c.4.5.1 1.5.1 1.5.7-.4.9-1.5.2-1.9zm7.8-7.2c-.4 1.1-1.2 2-2.2 2.5l-1.8-1c-.7-.4-.6-1.5.1-1.9l3.6-2c.2-.1.5 0 .6.2v.2a7 7 0 0 1-.3 2z"/><path d="M16.7 17c0 .6.3 1.2.8 1.5l9.9 5.7a9.6 9.6 0 0 0-4.7-7.9l-4.4-2.6c-.7-.4-.6-1.5.1-1.9l-1.1.6c-.3.2-.6.6-.6 1V17z"/><path d="m18.3 13.8 1.8 1-1.5-1.4c-.4-.5-.1-1.5-.1-1.5-.8.3-.9 1.4-.2 1.9z"/></svg>
```

File: entries/c/coindcx.com.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "CoinDCX": {
+      "domain": "coindcx.com",
+      "tfa": [
+        "sms",
+        "email",
+        "totp"
+      ],
+      "documentation": "https://support.coindcx.com/hc/en-gb/articles/360003181855-How-to-secure-your-account-with-Two-factor-Authentication-2FA-",
```
Comment (Carlgo11) [inline entries/c/coindcx.com.json]: The text after the numbers in the URL isn't needed.
```suggestion
      "documentation": "https://support.coindcx.com/hc/en-gb/articles/360003181855",
```

File: entries/c/coindcx.com.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      "keywords": [
+        "cryptocurrencies"
+      ]
```
Comment (Carlgo11) [inline entries/c/coindcx.com.json:15]: Looks like the products are mainly targeted to India
```suggestion
      ],
      "regions": [
        "in"
      ]
```

**Review by Carlgo11 — APPROVED** [review]

File: img/c/coindcx.com.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 32 32"><circle cx="16" cy="16" r="16" fill="#fff"/><path d="M10.5 20.9c.4-1.1 1.2-2 2.2-2.5l1.8 1c.7.4.6 1.5-.1 1.9l-3.6 2.1c-.2.1-.5 0-.6-.2V23c-.1-.7 0-1.4.3-2.1z"/><path d="M16.1 16.1c0-.6-.3-1.2-.8-1.5L5.4 8.9a9.6 9.6 0 0 0 4.7 7.9l4.4 2.6c.7.4.6 1.5-.1 1.9l1.1-.6c.3-.2.6-.6.6-1v-3.6z"/><path d="m14.5 19.4-1.8-1 1.5 1.4c.4.5.1 1.5.1 1.5.7-.4.9-1.5.2-1.9zm7.8-7.2c-.4 1.1-1.2 2-2.2 2.5l-1.8-1c-.7-.4-.6-1.5.1-1.9l3.6-2c.2-.1.5 0 .6.2v.2a7 7 0 0 1-.3 2z"/><path d="M16.7 17c0 .6.3 1.2.8 1.5l9.9 5.7a9.6 9.6 0 0 0-4.7-7.9l-4.4-2.6c-.7-.4-.6-1.5.1-1.9l-1.1.6c-.3.2-.6.6-.6 1V17z"/><path d="m18.3 13.8 1.8 1-1.5-1.4c-.4-.5-.1-1.5-.1-1.5-.8.3-.9 1.4-.2 1.9z"/></svg>
```
Comment (phallobst) [inline img/c/coindcx.com.svg]: This logo seems to have been phased out, in favor of the simple logotype:

**Review by phallobst — APPROVED** [review]

**Pull request merged**